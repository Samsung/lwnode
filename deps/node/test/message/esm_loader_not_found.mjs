// Flags:  --experimental-loader i-dont-exist
import '../common/index.mjs';
console.log('This should not be printed');
