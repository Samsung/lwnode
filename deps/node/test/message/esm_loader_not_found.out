(node:*) ExperimentalWarning: --experimental-loader is an experimental feature. This feature could change at any time
(Use `* --trace-warnings ...` to show where the warning was created)
internal/process/esm_loader.js:*
    internalBinding('errors').triggerUncaughtException(
                              ^
Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'i-dont-exist' imported from *
    at packageResolve (internal/modules/esm/resolve.js:*:*)
    at moduleResolve (internal/modules/esm/resolve.js:*:*)
    at Loader.defaultResolve [as _resolve] (internal/modules/esm/resolve.js:*:*)
    at Loader.resolve (internal/modules/esm/loader.js:*:*)
    at Loader.getModuleJob (internal/modules/esm/loader.js:*:*)
    at Loader.import (internal/modules/esm/loader.js:*:*)
    at internal/process/esm_loader.js:*:*
    at initializeLoader (internal/process/esm_loader.js:*:*)
    at Object.loadESM (internal/process/esm_loader.js:*:*)
    at runMainESM (internal/modules/run_main.js:*:*) {
  code: 'ERR_MODULE_NOT_FOUND'
}
